test/core/util/trickle_endpoint.cc

```diff
@@ -132,7 +132,7 @@ static int te_get_fd(grpc_endpoint* ep) {
 }
 
 static void te_finish_write(void* arg, grpc_error* error) {
-  trickle_endpoint* te = static_cast<trickle_endpoint*>(arg);
+  trickle_endpoint* te = reinterpret_cast<trickle_endpoint*>(arg);
```

Comment:
If `static_cast` was compiling here, why switch to `reinterpret_cast`?

Reply:
Oops, I slipped. Thanks for noticing.

Reply:
Specifically, my method here was to `s/reinterpret_cast/static_cast/g`, get all the build errors and change those back with an Emacs macro. Looks like I slipped here. I'll check the others.